proposals/0387-cross-compilation-destinations.md

```diff
@@ -80,85 +86,93 @@ The proposed structure of artifact bundles containing CC destinations looks like
 <name>.artifactbundle
 ├ info.json
 ├ <destination artifact>
-│ ├ <host variant>
-│ │ ├ destination.json
-│ │ └ <destination file tree>
-│ └ <host variant>
-│   ├ destination.json
-│   └ <destination file tree>
+│ ├ destination.json
+│ ├ toolset.json
+│ └ <destination files and directories>
 ├ <destination artifact>
-│ └ <host variant>
-│   ├ destination.json
-│   └ <destination file tree>
+│ ├ destination.json
+│ ├ toolset.json
+│ └ <destination files and directories>
 ├ <destination artifact>
 ┆ └┄
 ```
 
-For example, a destination bundle allowing to cross-compile Swift 5.7 source code to recent versions of Ubuntu from
+For example, a destination bundle allowing to cross-compile Swift 5.8 source code to recent versions of Ubuntu from
 macOS would look like this:
 
 ```
-swift-5.7_ubuntu.artifactbundle
+swift-5.8_ubuntu.artifactbundle
 ├ info.json
+├ toolset.json
 ├ ubuntu_jammy
-│ ├ arm64-apple-darwin
-│ │ ├ destination.json
+│ ├ destination.json
+│ ├ toolset.json
+│ ├ <destination files and directories shared between triples>
+│ ├ aarch64-unknown-linux-gnu
 │ │ ├ toolset.json
-│ │ └ <destination file tree>
-│ └ x86_64-apple-darwin
-│   ├ destination.json
+│ │ └ <triple-specific destination files and directories>
+│ └ x86_64-unknown-linux-gnu
 │   ├ toolset.json
-│   └ <destination file tree>
+│   └ <triple-specific destination files and directories>
 ├ ubuntu_focal
-│ └ x86_64-apple-darwin
-│   ├ destination.json
+│ ├ destination.json
+│ └ x86_64-unknown-linux-gnu
 │   ├ toolset.json
-│   └ <destination file tree>
+│   └ <triple-specific destination files and directories>
 ├ ubuntu_bionic
 ┆ └┄
 ```
 
-Here each artifact directory is dedicated to a specific CC destination, while binaries for a specific host platform are
-placed in `arm64-apple-darwin` and `x86_64-apple-darwin` subdirectories.
-
-`info.json` bundle manifests at the root of artifact bundles should specify `"type": "crossCompilationDestination"`
-for corresponding artifacts. Artifact identifiers in this manifest uniquely identify a CC destination. The rest of the
-properties of bundle manifests introduced in SE-0305 are preserved.
+Here each artifact directory is dedicated to a specific CC destination, while files specific to each triple are placed
+in `arm64-apple-darwin` and `x86_64-apple-darwin` subdirectories.
```

Comment:
Now that the host variant directories are removed, this should read "in `x86_64-unknown-linux-gnu` and `aarch64-unknown-linux-gnu` subdirectories" instead.